jdaviz/app.py

```diff
@@ -1135,9 +1140,7 @@ def get_sub_regions(self, subset_state, simplify_spectral=True,
                     else:
                         if isinstance(two, list):
                             two[0]['glue_state'] = "AndNotState"
-                        # Return two first so that we preserve the chronology of how
```

Comment:
Not sure what this comment is about because when you add two SpectralRegion, it automatically reorders by ascending wavelength, so it is never chronological to begin with. That is why this comment is removed.